sequence/sequence_manager.py

```diff
@@ -0,0 +1,27 @@
+from sequence.sequence_factory import get_sequence_generator
+
+
+class IdGenerator:
```

Comment:
these methods no longer work because self.table_name is undefined. I'd also recommend:

```
TABLE_NAME = 'the-table-name'

def __init__(self):
  self.sequence_generator = get_sequence_generator(IdGenerator.TABLE_NAME)
```

instead of creating a new instance in each method

Reply:
done